minio/api.py

```diff
@@ -142,6 +147,7 @@ class Minio(object):
     def __init__(self, endpoint, access_key=None,
                  secret_key=None,
                  session_token=None,
+                 credentials=None,
```

Comment:
This won't be a breaking change @vadmeste in python this is automatically optional.

Reply:
> This won't be a breaking change @vadmeste in python this is automatically optional.

It is for users who initialized Minio without keywords in arguments

Reply:
What @vadmeste means is it used thus:
```
minio = Minio("http://someserver", None, None, False)
```
this will fail. So put `credentials` last in the argument list (after `http_client`)

Reply:
Done, credentials parameter moved to end of list.